plugin-build/gradle.properties

```
@@ -6,4 +6,19 @@ version = 1.7.37-SNAPSHOT
 org.gradle.daemon=false
 
 # for debugging
-# org.gradle.logging.level=info
\ No newline at end of file
+# org.gradle.logging.level=info
+
+# publication pom properties
+POM_NAME=Sentry Android Gradle Plugin
+POM_DESCRIPTION=Sentry Android Gradle Plugin
+POM_URL=https://github.com/getsentry/sentry-android-gradle-plugin
+POM_SCM_URL=https://github.com/getsentry/sentry-android-gradle-plugin
+POM_SCM_CONNECTION=scm:git:git://github.com/getsentry/sentry-android-gradle-plugin.git
+POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/getsentry/sentry-android-gradle-plugin.git
+
+POM_LICENCE_NAME=MIT
+POM_LICENCE_URL=http://www.opensource.org/licenses/mit-license.php
```

What about changing the default branch to `main` when swap to the new code base?
Good chance to do that.

Meanwhile: 
```suggestion
POM_LICENCE_URL=https://github.com/getsentry/sentry-android-gradle-plugin/blob/master/LICENSE
```

---

Yup absolutely 👌